android-sdk/src/main/java/co/elastic/apm/android/sdk/features/persistence/PersistenceConfiguration.java

```diff
@@ -57,7 +63,19 @@ public Builder setMaxCacheSize(int maxCacheSize) {
             return this;
         }
 
+        /**
+         * Sets a scheduler that will take care of periodically read data stored in disk and
+         * export it.
+         */
+        public Builder setExportScheduler(ExportScheduler exportScheduler) {
+            this.exportScheduler = exportScheduler;
+            return this;
+        }
+
         public PersistenceConfiguration build() {
+            if (exportScheduler == null) {
+                exportScheduler = ExportScheduler.getDefault(60 * 60 * 1000); // Every hour by default
```

Comment:
Is the default for exporting data to the collector 60 minutes? 60 seconds should be more appropriate.

Reply:
I don't have a strong opinion about this, so if the iOS agent uses 60 seconds already then I'll do the same here to keep consistency!

Reply:
It's updated now